buildSrc/src/main/kotlin/non-multiplatform.publish-conventions.gradle.kts

```
@@ -0,0 +1,31 @@
+plugins {
+//     Regular java modules need 'java-library' plugin for proper publication
+    `java-library`
+    id("publish-conventions")
+}
+
+// MPP projects pack their sources automatically, java libraries need to explicitly pack them
+val sourcesJar = tasks.register<Jar>("sourcesJar") {
```

```suggestion
val sourcesJar by tasks.registering(Jar::class) {
```

using `by tasks.registering` means the name of the task is set once, in the property name, rather than using a string.

---

rewrote